src/tfr_navigation/src/navigation_action_client.cpp

```diff
@@ -14,11 +15,11 @@ int main(int argc, char** argv)
 
     NavigationClient client("navigation_action_server");
     //test preemption
-    client.navigate(tfr_msgs::NavigationGoal::TO_MINING);
-    client.stop_all();
+    client.navigate(tfr_msgs::LocationCode::MINING);
+//    client.stop_all();
```

Comment:
Commented out code should be removed.

Reply:
I don't agree with you in this context, It's a test class, that is on the chopping block to be removed in system integration. Next week my job is to literally throw out this code and incorperate the major responsibilities into executive.